### PR TITLE
Add new special collections locations

### LINKED ISSUE #1963: update bibdata with new locations from Alma
The following 9 locations were created; they are all Special Collections locations with reading room use only. 

NB these are also the locations for which we want to suppress the Reading Room Request button (https://github.com/pulibrary/orangelight/issues/2910).

rare$xmr
Fulfillment unit: Closed
Remote Storage: ReCAP remote storage

rare$scactsn
Fulfillment unit: Limited

rare$scagax
Fulfillment unit: Limited

rare$scamss
Fulfillment unit: Limited

rare$scawa
Fulfillment unit: Limited

rare$scaex
Fulfillment unit: Limited

rare$scahsvm
Fulfillment unit: Limited

mudd$scamudd
Fulfillment unit: Limited

rare$scathx
Fulfillment unit: Limited

Additional background can be found here:
[Project overview](https://docs.google.com/document/d/1FYpmZwRNuhnyoG2LfRGVm0nHyOhpgASYhFiV3lwOPXw/edit)
[Locations proposal](https://docs.google.com/document/d/1y__OO9UPU6tbUT42ijKxa_LlR5rRXg8-/edit)
[Locations checklist](https://docs.google.com/document/d/1E36NShlpqeq7zFXmMcP1Tu5ZD2NWgySgp20teTGiQWE/edit#)

### PR DESCRIPTION
**Pull request by christinach**
closes #1963

### CODE REVIEW

**Comment by christinach** [comment]
@mzelesky @kevinreiss can you check my question in this comment https://github.com/pulibrary/bibdata/issues/1963#issuecomment-1228926606 ?

**Comment by christinach** [comment]
The new locations are in https://bibdata-qa.princeton.edu/locations/holding_locations

File: marc_to_solr/translation_maps/location_display.rb.tmpl [hunk 1]
```diff
@@ -21,6 +20,8 @@
  "eastasian$ref"=>"East Asian Library",
  "eastasian$reserve"=>"East Asian Library",
  "eastasian$UNASSIGNED"=>"East Asian Library",
+ "online$etasrcp"=>"Electronic Access",
```
Comment (kevinreiss) [inline marc_to_solr/translation_maps/location_display.rb.tmpl:23]: ETAS came back?

Comment (kevinreiss) [reply marc_to_solr/translation_maps/location_display.rb.tmpl:23]: @mzelesky do we need this here for some legacy purpose? I can't think on any.

Comment (christinach) [reply marc_to_solr/translation_maps/location_display.rb.tmpl:23]: I'll remove it. It appears in this file because it is still in Alma.

**Comment by mzelesky** [comment]
Until about 5 minutes ago, there was still an item in an ETAS location. But now all the ETAS items are gone. I'll see if I can delete the locations entirely.

**Comment by mzelesky** [comment]
Success. The ETAS locations are gone from Alma.

**Comment by regineheberlein** [comment]
@christina we changed one label this morning: 
```
scahsvm
Internal name: Manuscripts High Security Archival
External name: Manuscripts Archival. Special Collections Use Only
```

Sorry for the late notice, and thank you!

**Review by kevinreiss — APPROVED** [review]

**Pull request merged**